doc/sphinxext/numpy_ext/docscrape_sphinx.py

```diff
@@ -27,7 +27,6 @@ def _str_indent(self, doc, indent=4):
         return out
 
     def _str_signature(self):
-        return ['']
```

Comment:
Could you probe a little on why this was introduced (using git blame) and whether this change of yours would alter any functionality?